fota/full.yaml

```diff
@@ -0,0 +1,20 @@
+# Code generated by moulin. All manual changes will be lost
+type: custom_script
```

Comment:
Where is the main moulin yaml file?

Reply:
Since we didn't use moulin to build AosEdge image, so there is no main moulin yaml.

Reply:
Please use Moulin to build the image

Reply:
Ok, we will create a Moulin file to build the image.